web/client/components/data/query/__tests__/GeometryDetails-test.jsx

```diff
@@ -172,5 +172,9 @@ describe('GeometryDetails', () => {
         let panelBodyRows = pb.getElementsByClassName('row');
         expect(panelBodyRows).toExist();
         expect(panelBodyRows.length).toBe(3);
+
+        [...panelBodyRows].forEach(row => {
```

Comment:
please refactor this a bit with something like 
const inputFields = someNode.document.querySelectorAll("input")
expect(inputFields.length).toBe(4);

please add a comment in the test explaining that since the extent is reprojected to 4326 it can go have 9 to 10 symbols (lon values can have 3 integers 123.123456)
maybe add the correspondi value in 4326